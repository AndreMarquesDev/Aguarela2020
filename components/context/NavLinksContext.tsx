import { createContext, Context } from 'react';

export interface NavLinksContextProps {
    setNavHeight?: React.Dispatch<React.SetStateAction<number>>;
}

const NavLinksContext: Context<NavLinksContextProps> = createContext(null);

export default NavLinksContext;
