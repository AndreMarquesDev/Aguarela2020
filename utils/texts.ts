/* eslint-disable line-comment-position */

export interface TextsInterface {
    about: string;
    projects: string;
    services: string;
    contact: string;
    welcome1: string;
    welcome2: string;
    welcome3: string;
    footerInfo: string;
    managingsocialMedia: string;
    throughStrategicPlanning: string;
    monitorAndOptimizeProcessAndStrategy: string;
    skills: string;
    socialMediaStrategy: string;
    makingAuditsAndAnalysis: string;
    socialMediaConsulting: string;
    weCanHelpYourTeam: string;
    communityManagement: string;
    whenWeSendAMessage: string;
    paidSocialAndSearch: string;
    planningAndImplementing: string;
    optimizationAndAnalysis: string;
    measuringResults: string;
    contentCreation: string;
    createAttractiveContent: string;
    workflow: string;
    defineTarget: string;
    defineGoals: string;
    platformStrategy: string;
    paidSocialAndInfluencers: string;
    insightsAndReports: string;
    optimization: string;
    whatIsTheTarget: string;
    whatAreYourGoals: string;
    whatPlatforms: string;
    relevantContentCreation: string;
    promotePostsAndCollaborate: string;
    detailedAnalysis: string;
    improvementsAndUpdatesToTheStrategy: string;
    socialMediaManagementAndContentCreation: string;
    present: string;
    inPartnershipWith: string;
    seeMore: string;
    letsWork: string;
    letsWorkDescription: string;
    myNetwork: string;
    hiMyNameIs: string;
    withMoreThan: string;
    thesocialMediaCommunicationStrategy: string;
    getToKnowMeBetter: string;
    contactMe1: string;
    contactMe2: string;
    contactMe3: string;
    needHelpWithYourBusiness: string;
    sendMeAnEmail: string;
    iAmAvailableToAdvise: string;
    design: string;
    socialMediaEn: string;
    digital: string;
    webDesign: string;
    outdoorsAndFlyers: string;
    contactCard: string;
    socialMediaTemplates: string;
    menus: string;
    marketAnalysis: string;
    socialMediaStrategyAndManagement: string;
    paidSocial: string;
    consulting: string;
    photographyAndVideo: string;
    paidSearchCampaigns: string;
    opinionPlatformsManagement: string;
    seoOptimization: string;
    sponsoredAds: string;
    socialMedia: string;
    consultingAndContentCreation: string;
    socialMediaAndContentCreation: string;
    socialMediaAndPaidSocial: string;
    socialMediaManagement: string;
    name: string;
    brandBusiness: string;
    email: string;
    subject: string;
    message: string;
    send: string;
    messageSentSuccessfully: string;
    thereWasAnErrorSendingTheMessage: string;
    pleaseEnterfirstAndLastName: string;
    pleaseEntervalidEmailAddress: string;
    pleaseEnterAMessage: string;
}

export const textsPt: TextsInterface = {
    about: 'Sobre',
    projects: 'Projetos',
    services: 'Serviços',
    contact: 'Contacto',
    welcome1: 'Bem',
    welcome2: 'Vin',
    welcome3: 'Dos',
    footerInfo: 'Aguarela Digital - Developed by',
    managingsocialMedia: 'Gestão de redes sociais está no meu ADN. Sou uma freelancer, baseada em Lisboa, com experiência em gestão de redes sociais e criação de conteúdo. A <a href="https://www.instagram.com/aguareladigital" target="_blank" rel="noreferrer" class="link">@Aguarela</a> surgiu com o objetivo de ajudar negócios locais e pequenas empresas a posicionarem-se no digital, gerar awareness, criar uma relação com o target e impulsionar vendas.',
    throughStrategicPlanning: 'Através do planeamento estratégico, criação de conteúdo e implementação estou disponível para ser o vosso braço direito.',
    monitorAndOptimizeProcessAndStrategy: 'Do ínicio ao fim, acompanho, monitorizo e otimizo todo o processo e estratégia de gestão de redes sociais.',
    skills: 'Skills',
    socialMediaStrategy: 'Estratégia de Redes Sociais',
    makingAuditsAndAnalysis: 'Realizar uma auditoria e análise detalhada, com base nos insights e objetivos da marca, definindo em que redes sociais a marca deve estar presente e quais as oportunidades disponíveis.',
    socialMediaConsulting: 'Consultoria de Redes Sociais',
    weCanHelpYourTeam: 'Posso ajudar a sua equipa na forma como a marca está presente nas redes sociais, como comunica, como alcançar os objetivos pretendidos e como aproveitar as oportunidades disponíveis no mercado digital do seu negócio.',
    communityManagement: 'Gestão de Comunidade',
    whenWeSendAMessage: 'Quando enviamos uma mensagem aos nossos amigos e eles demoram dias a responder-nos não é muito agradável e o mesmo acontece com as marcas. Faço gestão de comunidade, respondendo a mensagens e comentários da sua audiência em prazos toleráveis e ideais.',
    paidSocialAndSearch: 'Paid Social e Search',
    planningAndImplementing: 'Planeio, implemento e acompanho campanhas paid social (redes sociais) e campanhas paid search (anúncios no Google), otimizando os objetivos de marketing da sua marca ou negócio.',
    optimizationAndAnalysis: 'Análise e Otimização',
    measuringResults: 'Medir resultados de forma a otimizar a estratégia é a chave para uma presença de sucesso nas redes sociais. Em todos os meus projetos faço uma análise mensal detalhada de forma a existir uma adaptação contínua ao mercado e às exigências dos consumidores.',
    contentCreation: 'Criação de Conteúdo',
    createAttractiveContent: 'Crio conteúdo (fotográfico, animado, design e vídeo) atrativo e relevante para os consumidores. Ao reunir insights de consumidores e know-how consigo saber o tipo de conteúdo que melhor funciona para cada negócio.',
    workflow: 'Workflow',
    defineTarget: 'Definir target',
    defineGoals: 'Definir objetivos',
    platformStrategy: 'Estratégia de plataformas',
    paidSocialAndInfluencers: 'Paid social & influencers',
    insightsAndReports: 'Insights e reports',
    optimization: 'Otimização',
    whatIsTheTarget: 'Quem é o público-alvo da marca',
    whatAreYourGoals: 'Quais são os objetivos de negócio e comunicação',
    whatPlatforms: 'Quais as plataformas onde a marca deve estar presente',
    relevantContentCreation: 'Criação de conteúdo relevante e de qualidade para o target',
    promotePostsAndCollaborate: 'Promoção de posts e colaborações com influencers',
    detailedAnalysis: 'Análises detalhadas e identificação de insights',
    improvementsAndUpdatesToTheStrategy: 'Melhorias e updates à estratégia',
    socialMediaManagementAndContentCreation: 'Gestão de Redes Sociais e Criação de Conteúdo',
    present: 'Presente',
    inPartnershipWith: 'em dupla com Sofia Ferreirinho',
    seeMore: 'Ver mais',
    letsWork: 'Vamos trabalhar juntos!',
    letsWorkDescription: 'Precisa de ajuda com as redes sociais e marketing digital da sua marca ou negócio? Envie-me um email e vamos conversar. Estou disponível para ajudar a posicionar a sua marca no mundo digital.',
    myNetwork: 'A minha rede',
    hiMyNameIs: 'Olá! O meu nome é Catarina Santiago e sou gestora de redes sociais e criadora de conteúdo baseada em Lisboa.',
    withMoreThan: 'Com mais de 3 anos e 18 clientes de experiência, a Aguarela surgiu com a missão de providenciar um serviço premium de gestão de redes sociais e criação de conteúdo a negócios locais e pequenas empresas.',
    thesocialMediaCommunicationStrategy: 'A estratégia de comunicação em redes sociais é sempre desenvolvida com base nos objetivos do cliente. Embora seja importante ser flexível e compreensiva, o meu papel é pensar fora da caixa, criar conteúdo inovador para redes sociais e otimizar a estratégia com base nos resultados. Acredito que uma boa estratégia de comunicação aliada a um planeamento visual de conteúdo é o segredo para o aumento da visibilidade e faturação do negócio.',
    getToKnowMeBetter: 'Conhece-me melhor',
    contactMe1: 'Con',
    contactMe2: 'Ver',
    contactMe3: 'Sar',
    needHelpWithYourBusiness: 'Precisa de ajuda com o seu negócio, projeto ou marca?',
    sendMeAnEmail: 'Envie-me um email e vamos conversar. Podemos começar o quanto antes a definir uma estratégia para social media de acordo com os seus objetivos.',
    iAmAvailableToAdvise: 'Estou ainda disponível para dar consultoria em social media e partilhar todos os passos e dicas de que precisa para cumprir os seus objetivos.',
    design: 'Design',
    socialMediaEn: 'Social Media',
    digital: 'Digital',
    webDesign: 'Web design',
    outdoorsAndFlyers: 'Cartazes & flyers',
    contactCard: 'Cartão de contacto',
    socialMediaTemplates: 'Templates para Social Media',
    menus: 'Menus',
    marketAnalysis: 'Análises de mercado',
    socialMediaStrategyAndManagement: 'Estratégia e gestão de redes sociais',
    paidSocial: 'Promoção de posts',
    consulting: 'Consultoria',
    photographyAndVideo: 'Fotografia e vídeo',
    paidSearchCampaigns: 'Campanhas paid search (google ads)',
    opinionPlatformsManagement: 'Gestão de plataformas de opinião',
    seoOptimization: 'Otimização (SEO)',
    sponsoredAds: 'Anúncios patrocinados (LinkedIn Ads, Google Ads e Facebook Ads)',
    socialMedia: 'Redes sociais',
    consultingAndContentCreation: 'Consultoria e criação de conteúdo',
    socialMediaAndContentCreation: 'Redes sociais e criação de conteúdo',
    socialMediaAndPaidSocial: 'Redes sociais e paid social',
    socialMediaManagement: 'Gestão de redes sociais',
    name: 'Nome',
    brandBusiness: 'Marca/Negócio',
    email: 'Email',
    subject: 'Assunto',
    message: 'Mensagem',
    send: 'Enviar',
    messageSentSuccessfully: 'Mensagem enviada com sucesso.',
    thereWasAnErrorSendingTheMessage: 'Ocorreu um erro no envio da mensagem. Tente outra vez mais tarde ou entre em contacto através do meu email <a href="mailto:catarina@aguareladigital.pt" class="link">catarina@aguareladigital.pt</a>',
    pleaseEnterfirstAndLastName: 'Este campo é obrigatório. Por favor insira o seu primeiro e último nome',
    pleaseEntervalidEmailAddress: 'Este campo é obrigatório. Por favor insira um endereço de email válido',
    pleaseEnterAMessage: 'Este campo é obrigatório. Por favor insira uma mensagem',
};

export const textsEn: TextsInterface = {
    about: 'About',
    projects: 'Projects',
    services: 'Services',
    contact: 'Contact',
    welcome1: 'Wel',
    welcome2: 'Co',
    welcome3: 'Me',
    footerInfo: 'Aguarela Digital - Developed by',
    managingsocialMedia: 'Social media management is in my DNA. I\'m a freelancer, based in Lisbon, with experience in social media management and content creation. <a href="https://www.instagram.com/aguareladigital" target="_blank" rel="noreferrer" class="link">@Aguarela</a>\'s goal is to help local businesses and small companies position themselves digitally, create awareness, create a realtionship with the target and boost sales.',
    throughStrategicPlanning: 'Through strategic planning, content creation and implementation I can be your right hand.',
    monitorAndOptimizeProcessAndStrategy: 'From start to finish I keep up with, monitor and optimize the whole social media management process and strategy.',
    skills: 'Skills',
    socialMediaStrategy: 'Social Media strategy',
    makingAuditsAndAnalysis: 'Making audits and detailed analysis, based on the brand\'s insights and goals, deciding in what social networks the brand should be present in and what are the available opportunities.',
    socialMediaConsulting: 'Social Media consulting',
    weCanHelpYourTeam: 'I can help your team in the way your brand is present on social media, how it communicates, how to achieve the intended goals and how to take advantage of the opportunities available in the digital market of your business.',
    communityManagement: 'Community Management',
    whenWeSendAMessage: 'When we send a message to our friends and they take days to answer us it is not very pleasant and the same is true with brands. I do community management, responding to messages and comments from your audience within tolerable and ideal deadlines.',
    paidSocialAndSearch: 'Paid Social and Search',
    planningAndImplementing: 'I plan, implement and monitor paid social campaigns (social media) and paid search campaigns (ads on Google), optimizing the marketing objectives of your brand or business.',
    optimizationAndAnalysis: 'Optimization and Analysis',
    measuringResults: 'Measuring results in order to optimize the strategy is the key to a successful presence on social media. In all my projects I do a detailed monthly analysis so that there is a continuous adaptation to the market and to the demands of consumers.',
    contentCreation: 'Content creation',
    createAttractiveContent: 'I create attractive and relevant content to consumers (photographic, animated, design and video). By gathering consumer insights and know-how, I know the type of content that works best for each business.',
    workflow: 'Workflow',
    defineTarget: 'Set the target',
    defineGoals: 'Set goals',
    platformStrategy: 'Platforms strategies',
    paidSocialAndInfluencers: 'Paid social & influencers',
    insightsAndReports: 'Insights and reports',
    optimization: 'Optimization',
    whatIsTheTarget: 'What\'s the target',
    whatAreYourGoals: 'What are the business and communication goals',
    whatPlatforms: 'In what platforms should the brand be present',
    relevantContentCreation: 'Relevant and quality content created for the target',
    promotePostsAndCollaborate: 'Promoted posts and collaborations with influencers',
    detailedAnalysis: 'Detailed analysis and insights\' finding',
    improvementsAndUpdatesToTheStrategy: 'Strategy updates and improvements',
    socialMediaManagementAndContentCreation: 'Social Media Management and Content Creation',
    present: 'Present',
    inPartnershipWith: 'in partnership with Sofia Ferreirinho',
    seeMore: 'See more',
    letsWork: 'Let\'s work together!',
    letsWorkDescription: 'Need help with digital marketing and the social media of your brand or business? Send me an email and let\'s talk. I\'m available to help position your brand in the digital world.',
    myNetwork: 'My network',
    hiMyNameIs: 'Hi! My name is Catarina Santiago and I\'m a social media manager and content creator based in Lisbon.',
    withMoreThan: 'With more than 3 years of experience and 18 clients, Aguarela was created with the mission to provide a premium service of social media management and content creation to local businesses and small companies.',
    thesocialMediaCommunicationStrategy: 'The social media communication strategy is always developed based on the client\'s goals. Even though it is important to be flexible and comprehensive, my role is to think outside the box, create inovative content and optimize the strategy based on results. I believe that a good communication strategy together with a visual planning of the content is the secret to increase the visibility of the business.',
    getToKnowMeBetter: 'Get to know me better',
    contactMe1: 'Con',
    contactMe2: 'Tac',
    contactMe3: 'Tme',
    needHelpWithYourBusiness: 'Need help with your business, project or brand?',
    sendMeAnEmail: 'Send me an email and let\'s talk. We can start as soon as possible defining a social media strategy that meets your objectives.',
    iAmAvailableToAdvise: 'I am also available to provide social media consulting and share all the tips you need to fulfill your goals.',
    design: 'Design',
    socialMediaEn: 'Social Media',
    digital: 'Digital',
    webDesign: 'Web design',
    outdoorsAndFlyers: 'Outdoors & flyers',
    contactCard: 'Contact card',
    socialMediaTemplates: 'Social Media templates',
    menus: 'Menus',
    marketAnalysis: 'Market analysis',
    socialMediaStrategyAndManagement: 'Social Media Strategy',
    paidSocial: 'Paid social',
    consulting: 'Consulting',
    photographyAndVideo: 'Photography and video',
    paidSearchCampaigns: 'Paid search campaigns (google ads)',
    opinionPlatformsManagement: 'Opinion platforms management',
    seoOptimization: 'SEO optimization',
    sponsoredAds: 'Sponsored ads (LinkedIn Ads, Google Ads e Facebook Ads)',
    socialMedia: 'Social media',
    consultingAndContentCreation: 'Consulting and content creation',
    socialMediaAndContentCreation: 'Social media and content creation',
    socialMediaAndPaidSocial: 'Social media and paid social',
    socialMediaManagement: 'Social media management',
    name: 'Name',
    brandBusiness: 'Brand/Business',
    email: 'Email',
    subject: 'Subject',
    message: 'Message',
    send: 'Send',
    messageSentSuccessfully: 'Message sent successfully.',
    thereWasAnErrorSendingTheMessage: 'Thre was an error sending your message. Please try again later ou contact me through my email <a href="mailto:catarina@aguareladigital.pt" class="link">catarina@aguareladigital.pt</a>',
    pleaseEnterfirstAndLastName: 'This field is required. Please enter your first and last name',
    pleaseEntervalidEmailAddress: 'This field is required. Please enter a valid email address',
    pleaseEnterAMessage: 'This field is required. Please enter a message',
};
