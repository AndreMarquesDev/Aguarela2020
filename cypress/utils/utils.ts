import { Locale } from '../../types/Locale';
import {
    menuIconButtonDataTestId,
    nukaCarouselNextButtonDataTestId,
} from '../../utils/dataTestIds';
import { TextsInterface, textsPt, textsEn } from '../../utils/texts';

export const getLocalizedTexts = (locale: Locale): TextsInterface => {
    if (locale === Locale.Pt) {
        return textsPt;
    }

    return textsEn;
};

export const openMenuMobile = (isMobile: boolean, isFirefox: boolean): void => {
    if (isMobile) {
        cy.getByDataTestId(menuIconButtonDataTestId).click({ force: isFirefox });
    }
};

export const clickNextArrowButtonIfMobile = (
    isMobile: boolean,
    container: string,
    containerTitle: string
): void => {
    const isFirefox = Cypress.isBrowser('firefox');

    if (isMobile) {
        cy.getByDataTestIdParent(container, nukaCarouselNextButtonDataTestId).click({
            force: isFirefox,
        });

        cy.getByText(container, containerTitle).scrollIntoView();
    }
};
