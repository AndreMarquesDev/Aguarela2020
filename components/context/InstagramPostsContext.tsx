import { createContext, Context } from 'react';
import { IInstagramPost } from '../../utils/generic';

export interface IInstagramPostsContext {
    posts: IInstagramPost[];
}

const InstagramPostsContext: Context<IInstagramPostsContext> = createContext(null);

export default InstagramPostsContext;
