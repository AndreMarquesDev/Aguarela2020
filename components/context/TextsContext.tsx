import { createContext, Context } from 'react';
import { TextsInterface, textsPt } from '../../utils/texts';

export interface TextsContextProps {
    texts: TextsInterface;
}

export const TextsContext: Context<TextsContextProps> = createContext({
    texts: textsPt,
});
