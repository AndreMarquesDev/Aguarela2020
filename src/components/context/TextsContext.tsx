import type { Context } from 'react';
import type { TextsInterface } from '../../utils/texts';
import { createContext } from 'react';
import { textsPt } from '../../utils/texts';

export type TextsContextProps = {
    texts: TextsInterface;
};

export const TextsContext: Context<TextsContextProps> = createContext({
    texts: textsPt,
});
