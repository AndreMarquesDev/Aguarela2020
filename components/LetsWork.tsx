import React, { FC, useContext } from 'react';
import Button from './Button';
import TextsContext from './context/TextsContext';
import Title from './Title/Title';

const LetsWork: FC = () => {
    const { texts } = useContext(TextsContext);

    return (
        <>
            <section className="container">
                <div className="wrapper genericMargins">
                    <Title text={texts.letsWork} />

                    <p className="bodyText bodyText--white">{texts.letsWorkDescription}</p>

                    <Button page="contact">{texts.contact}</Button>
                </div>
            </section>

            <style jsx>
                {`
                    @import './styles/_vars.scss';

                    .container {
                        background: $purple;

                        .wrapper {
                            max-width: 1060rem;
                        }
                    }
                `}
            </style>
        </>
    );
};

export default LetsWork;
