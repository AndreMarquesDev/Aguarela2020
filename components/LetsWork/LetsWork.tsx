import React, { useContext } from 'react';
import { letsWorkSectionDataTestId } from '../../utils/dataTestIds';
import { Button } from '../Button/Button';
import { TextsContext } from '../context/TextsContext';
import { Title } from '../Title/Title';

export const LetsWork = (): JSX.Element => {
    const { texts } = useContext(TextsContext);

    return (
        <>
            <section className="container" data-testid={letsWorkSectionDataTestId}>
                <div className="wrapper genericMargins">
                    <Title text={texts.letsWork} />

                    <p className="bodyText bodyText--white">{texts.letsWorkDescription}</p>

                    <Button page="contact">{texts.contact}</Button>
                </div>
            </section>

            <style jsx>
                {`
                    @import './styles/_vars.scss';

                    .container {
                        background: $purple;

                        .wrapper {
                            max-width: 1060rem;
                        }
                    }
                `}
            </style>
        </>
    );
};
