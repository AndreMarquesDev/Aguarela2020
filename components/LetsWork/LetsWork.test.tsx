import '@testing-library/jest-dom';
import React from 'react';
import { render, RenderResult, screen } from '@testing-library/react';
import { LetsWork } from './LetsWork';
import { TextsContext } from '../context/TextsContext';
import { textsEn, textsPt } from '../../utils/texts';

const renderComponent = (): RenderResult => {
    return render(<LetsWork />);
};

describe('<LetsWork />', () => {
    test('renders properly', () => {
        const { container } = renderComponent();

        expect(screen.getByText(textsPt.letsWork)).toBeInTheDocument();
        expect(screen.getByText(textsPt.letsWorkDescription)).toBeInTheDocument();
        expect(screen.getByText(textsPt.contact)).toBeInTheDocument();

        expect(container).toMatchSnapshot();
    });

    test('renders properly in English', () => {
        const { container } = render(
            <TextsContext.Provider
                value={{
                    texts: textsEn,
                }}
            >
                <LetsWork />
            </TextsContext.Provider>
        );

        expect(screen.getByText(textsEn.letsWork)).toBeInTheDocument();
        expect(screen.getByText(textsEn.letsWorkDescription)).toBeInTheDocument();
        expect(screen.getByText(textsEn.contact)).toBeInTheDocument();

        expect(container).toMatchSnapshot();
    });
});
