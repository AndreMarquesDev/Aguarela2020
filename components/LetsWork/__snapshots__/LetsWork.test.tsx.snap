// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LetsWork /> renders properly 1`] = `
<div>
  <section
    class="jsx-2032054637 container"
    data-testid="letsWork_section"
  >
    <div
      class="jsx-2032054637 wrapper genericMargins"
    >
      <h1
        class="jsx-3155756322 "
      >
        Vamos trabalhar juntos!
      </h1>
      <p
        class="jsx-2032054637 bodyText bodyText--white"
      >
        Precisa de ajuda com as redes sociais e marketing digital da sua marca ou negócio? Envie-me um email e vamos conversar. Estou disponível para ajudar a posicionar a sua marca no mundo digital.
      </p>
      <div
        class="jsx-781296904 buttonWrapper"
      >
        <div
          class="jsx-781296904 buttonBackground alignCenter"
        >
          <a
            class="jsx-781296904 button"
            href="/pt/contact"
          >
            Contacto
          </a>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`<LetsWork /> renders properly in English 1`] = `
<div>
  <section
    class="jsx-2032054637 container"
    data-testid="letsWork_section"
  >
    <div
      class="jsx-2032054637 wrapper genericMargins"
    >
      <h1
        class="jsx-3155756322 "
      >
        Let's work together!
      </h1>
      <p
        class="jsx-2032054637 bodyText bodyText--white"
      >
        Need help with digital marketing and the social media of your brand or business? Send me an email and let's talk. I'm available to help position your brand in the digital world.
      </p>
      <div
        class="jsx-781296904 buttonWrapper"
      >
        <div
          class="jsx-781296904 buttonBackground alignCenter"
        >
          <a
            class="jsx-781296904 button"
            href="/pt/contact"
          >
            Contact
          </a>
        </div>
      </div>
    </div>
  </section>
</div>
`;
